tests/ci-csi-cinder-e2e.sh

```diff
@@ -114,6 +114,11 @@ scp -i ~/.ssh/google_compute_engine \
   -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no \
   -r ${USERNAME}@${PUBLIC_IP}:/var/log/cinder-csi-e2e.log $ARTIFACTS/logs/cinder-csi-e2e.log || true
 
+# Fetch cinder-csi pod tests logs for debugging purpose
+scp -i ~/.ssh/google_compute_engine \
+  -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no \
+  -r ${USERNAME}@${PUBLIC_IP}:/var/log/csi-pod/* $ARTIFACTS/logs/ || true
```

Comment:
may be we could create folder /var/log/cinder-csi , and dump all the logs there , so that we can just copy the folder instead of multiple scp comands.

Reply:
done